kie-osgi/kie-karaf-features/src/main/filtered-resources/repository/features-core.xml

```diff
@@ -104,9 +104,9 @@
     <feature>camel-core</feature>
     <feature>camel-spring</feature>
     <feature>camel-cxf</feature>
-    <feature version="[3.3,3.5)">cxf-specs</feature>
-    <feature version="[3.3,3.5)">cxf-core</feature>
-    <feature version="[3.3,3.5)">cxf-jaxrs</feature>
+    <feature version="[3.3,3.6)">cxf-specs</feature>
+    <feature version="[3.3,3.6)">cxf-core</feature>
+    <feature version="[3.3,3.6)">cxf-jaxrs</feature>
```

Comment:
so if you extend the range for cxf-* deps, shouldn't be changed also kie-parent version property at https://github.com/kiegroup/droolsjbpm-build-bootstrap/commit/bbb34b2be08cedd09f93fcca7af0b8f2110dc473#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R316

Reply:
> so if you extend the range for cxf-* deps, shouldn't be changed also kie-parent version property at [kiegroup/droolsjbpm-build-bootstrap@bbb34b2#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R316](https://github.com/kiegroup/droolsjbpm-build-bootstrap/commit/bbb34b2be08cedd09f93fcca7af0b8f2110dc473#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R316)

@mareknovotny I'm afraid if I misunderstand your comment. You mean I should change the `droolsjbpm-build-bootstrap/pom.xml`? No, I changed the version range in the `features-core.xml` to meet the current `droolsjbpm-build-bootstrap/pom.xml` 's `version.org.apache.cxf` value `3.5.5` in order to pass the tests.

Reply:
yes, currently kie-parent pom has 3.5.5 and i saw that 3.6, but I didn't realize it is less than 3.6.